### DISCUSSION TITLE
howto digitally sign a PDF?

### DISCUSSION THREAD

**Discussion by steadfasterX** [JakubMelka/PDF4QT#162] [Q&A] [answered]
sorry for that stupid question but I am able to import my cert, verify certs but how to sign it actually with mine? Using linux 1.3.6-2 (arch linux)

**Comment by JakubMelka** [comment, marked as answer]
Hello @steadfasterX,
sorry for late response, I miss the discussion. Just activate the "Signature" plugin: Go to Options, select Signature plugin and restart the application:
![image](https://github.com/JakubMelka/PDF4QT/assets/11367048/4c4ba160-3992-4e5f-9f81-579992449086)

**Comment by steadfasterX** [comment]
omg yes! that was it.. not very intuitive ;) but glad it works now :) thanks for your reply
